### PR TITLE
Exposing backing web view if it's WKWebView

### PR DESCRIPTION
**Pull request by rztakashi**
*WKWebView wrapped inside poseur web view be exposed if the private header is used by the host app.
*Changed property qualifier from weak to strong.

### CODE REVIEW

**Comment by ZevEisenberg** [comment]
It looks like the permissions changed on `RZPoseurWebView_Private.h`. Was this intentional, and does it matter?

**Comment by rztakashi** [comment]
@ZevEisenberg , which permission are you referring to?

**Comment by ZevEisenberg** [comment]
@rztakashi 
permissions

**Comment by rztakashi** [comment]
Ah, good eyes, I didn't even realize that.  

The property that was added ideally shouldn't be widely used and hence was in a private header.  But it's required for UserTesting project so the code was changed in there first.  Then that file got copied over to RZPoseurWebView project since RZPoseurWebView now is not integrated into UserTesting workspace as subproject or submodule.  The permission change is carryover from me copying the modified file over from UserTesting to RZPoseurWebView.

So, to answer your question, no it was not intentional.  I don't think it affects the behavior any way.  @jkaufman , do you know?

**Comment by jkaufman** [comment]
Wow, good catch. It's setting the executable bit. While it _shouldn't_ matter, the value could keep flip flopping. Please reverse the change and consider [updating your git config](http://stackoverflow.com/questions/1257592) to avoid accidental permissions changes in the future.

**Comment by rztakashi** [comment]
@ZevEisenberg, @jkaufman
Fixed.

File: Classes/RZPoseurWebView.m [hunk 1]
```diff
@@ -39,7 +40,7 @@
 @interface RZPoseurWebView ()
 
 @property (strong, nonatomic, readwrite) NSURLRequest *request;
-
+@property (strong, nonatomic) RZPoseurWebView *instance;
```
Comment (jkaufman) [inline Classes/RZPoseurWebView.m]: Why add this property? Each subclass can reference `self.

**Comment by jkaufman** [comment]
+2, good addition, thank you.

**Pull request merged**